recipes-kernel/linux/linux-imx-headers_5.15.bb

```diff
@@ -39,17 +39,17 @@ IMX_UAPI_HEADERS = " \
 
 do_install() {
     # We install all headers inside of B so we can copy only the
-    # whitelisted ones, and there is no risk of a new header to be
+    # i.MX-specific ones, and there is no risk of a new header to be
     # installed by mistake.
     oe_runmake headers_install INSTALL_HDR_PATH=${B}${exec_prefix}
 
     # Kernel should not be exporting this header
-    rm -f ${D}${exec_prefix}/include/scsi/scsi.h
+    rm -f ${B}${exec_prefix}/include/scsi/scsi.h
 
     # The ..install.cmd conflicts between various configure runs
-    find ${D}${includedir} -name ..install.cmd | xargs rm -f
+    find ${B}${includedir} -name ..install.cmd | xargs rm -f
```

Comment:
@zandrey, these lines for `scsi.h` and `install.cmd` puzzled me, and I thought to remove them, but I saw that they were in `linux-libc-headers` and thought perhaps your intent was to copy what was there just in case? Seems a maintenance problem though keeping these in sync. Is it worth it? At the least I guess we need a comment?

Reply:
I think the closer it is, easier the maintenance. So it makes sense to let it aligned.